cluster/gce/manifests/e2e-image-puller.manifest

```diff
@@ -1,117 +0,0 @@
-# e2e-image-puller seeds nodes in an e2e cluster with test images.
-apiVersion: v1
-kind: Pod
-metadata:
-  name: e2e-image-puller
-  namespace: kube-system
-  labels:
-    name: e2e-image-puller
-spec:
-  containers:
-  - name: image-puller
-    resources:
-      requests:
-        cpu: 50m
-      limits:
-        cpu: 50m
-    image: k8s.gcr.io/busybox:1.24
-    # TODO: Replace this with a go script that pulls in parallel?
```

Comment:
Did we ever build anything like this? I'd like to do this for some test clusters that may not have the fastest image pull and I suspect this is causing flakes for them. 
If there's at least a good way to dump the list of images somewhere that'd be really helpful. If not, maybe we could add something to dump the images for consumption by testing setups? I'd be happy to work on this.